### PR TITLE
Setup the branding overrides for Ideasbox and Koombooks

### PR DESCRIPTION
**Pull request by bochecha**
New Ideascube versions now come with a default branding and a mechanism
to override it.

As a result, it becomes the responsibility of Ansiblecube to use that
new mechanism.

This commit installs the right branding in the correct folder, either
for Ideasbox or Koombook.

---

Do note that something is missing here: the right branding to deploy is governed by the `ideascube_branding` variable, but it isn't defined anywhere.

What we want is to have it defined by default to `ideasbox` for the IDBs, `koombook` for the KBs, and have the possibility for any box (or group of boxes) to set it to their own value.

For example, we could want to add a new branding folder for the Colombian « Biblioteca Móvil » project, and have that variable set to `bibliotecamovil` for all the servers which are part of this project.

I'm not sure what the best way to do this would be, though, so I'm happy to let one of the Ansible gurus take care of finishing this. :sweat_smile:

### CODE REVIEW

File: group_vars/x86_64.yml [hunk 1]
```diff
@@ -4,5 +4,6 @@ hostname: ideasbox.lan
 hotspot_name: ideasbox
 wpa_pass: idb258456
 ideascube_project_name: ideasbox
+ideascube_branding: ideasbox
```
Comment (bochecha) [inline group_vars/x86_64.yml:7]: Heh, that simple. Now I feel bad for not thinking about it. :stuck_out_tongue: 

Thanks!

File: roles/ideascube/tasks/branding.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+- include_vars: group_vars/{{ ansible_architecture }}.yml
+  tags: ['master', 'custom']
+
+- name: Install the branding resources
+  copy: src=branding/{{ ideascube_branding }}/ dest=/usr/share/ideascube/branding/
```
Comment (letompouce) [inline roles/ideascube/tasks/branding.yml]: `tags master custom`

File: roles/ideascube/tasks/branding.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+- include_vars: group_vars/{{ ansible_architecture }}.yml
+  tags: ['master', 'custom']
+
```
Comment (letompouce) [inline roles/ideascube/tasks/branding.yml:8]: You have to create the `/usr/share/ideascube/branding/` directory first. cf https://github.com/ideascube/ansiblecube/blob/oneUpdateFile/roles/system/tasks/main.yml#L284 for example.

Comment (bochecha) [reply roles/ideascube/tasks/branding.yml:8]: No need, it's now created by the ideascube package.

Comment (bochecha) [reply roles/ideascube/tasks/branding.yml:8]: Hmm, but the branding is done before installing the package. I guess the folder does need to be created then, doing it now.

Comment (bochecha) [reply roles/ideascube/tasks/branding.yml:8]: Done.

**Pull request merged**